src/NuProj.Targets/NuProj.targets

```diff
@@ -148,7 +150,7 @@
     <ProjectReference>
       <PackageDirectory>Lib</PackageDirectory>
       <TargetFramework></TargetFramework>
-      <Targets>$(ProjectOutputGroups)</Targets>
+      <ProjectOutputGroups>$(ProjectOutputGroups)</ProjectOutputGroups>
```

Comment:
The default value for `Targets` metadata is empty string. You won't find $(ProjectReferenceBuildTargets) defined anywhere in MSBuild directory by default. See `Microsoft.Common.CurrentVersion.targets` for yourself:
```xml
  <ItemDefinitionGroup>
    <ProjectReference>
      <!-- Target to build in the project reference; by default, this property is blank, indicating the default targets-->
      <Targets>$(ProjectReferenceBuildTargets)</Targets>
      <!-- Extra item type to emit outputs of the destination into. Defaults to blank. To emit only into this list, set the ReferenceOutputAssembly metadata to false as well. -->
      <OutputItemType/>
      <ReferenceSourceTarget>ProjectReference</ReferenceSourceTarget>
    </ProjectReference>
  </ItemDefinitionGroup>
```
We need `$(ProjectOutputGroups)` (or whatever do we want to call it), because we cannot use `ResolveAssemblyReferences` to get all required artifacts and dependencies, because we can target multiple different frameworks and we don't available have half of the item it needs.